README.md

```diff
@@ -1,2 +1,56 @@
+## Quick start
 1. Install Rust from https://rustup.rs/
 2. `cargo run`
+
+## About
+`solana-cli-template` is a sample app demonstrating the creation of a minimal CLI application written in Rust to interact with Solana.
+It provides three pieces of functionality:
+
+- `ping`: Creates a transaction sending 0 SOL from the signer's account to the signer's account. Returns the signature of the transaction.
+- `balance`: Returns the account's balance.
+- `help`: Tips for using the app. This is an off-chain operation.
+
+## Local step-by-step
+1. Preparep account:
+  - Start a local node: `solana test validator`.
+  - Generate a keypair: `solana-keygen new -o test.json`.
+  - Add 100 SOL to the corresponding account `solana airdrop --url http://127.0.0.1:8899 --keypair test.json 100`.
+
+2. Bulid app: `cargo run`.
+
+3. Ping:
+  ```
+  $ ./target/debug/cli-template ping --url http://127.0.0.1:8899 --keypair test.json
```

Comment:
```suggestion
  $ cargo run -- ping --url http://127.0.0.1:8899 --keypair test.json
```

Reply:
Cool, I didn't know this syntax. Adopted: https://github.com/mvines/solana-cli-template/pull/2/commits/cbf1094b7ff393c9de85c85a124eea68dbb611b8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R23